scripts/test-infra/test-e2e-presubmit.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+set -o errexit
+
+# Configure environment
+export IMAGE_REGISTRY="localhost:5001"
+export IMAGE_TAG_NAME=$(git describe --tags --dirty --always)
+export MANIFESTS_DIR="deployment/overlays/default"
```

Comment:
What is this for?

Reply:
sorry forgot to clean up after test, removing it...